src/components/PageElements/Elements/HistoryResponseElement.tsx

```diff
@@ -15,41 +16,50 @@ const HistoryResponseElement: React.FC<HistoryElement> = ({
   elementTypePluginCode,
   parameters,
 }) => {
+  const { strings } = useLanguageProvider()
   return (
     <div className="response-element-content">
       <p className="secondary author-name">{author}</p>
       <div className="decision-container">
-        <div className="comment-container">
-          <Header
-            as="h5"
-            // icon={attentionRequired && <Icon name="exclamation circle" color="red" size="tiny" />}
-            content={title}
-          />
-          <p className="secondary date-indicator">{getSimplifiedTimeDifference(timeUpdated)}</p>
-        </div>
-        {elementTypePluginCode ? (
-          <SummaryViewWrapper
-            element={
-              {
-                parameters,
-                pluginCode: elementTypePluginCode,
-                isRequired: true,
-                isVisible: true,
-              } as ElementState
-            }
-            response={response}
-            // Not passing any other reponses, as these elements will be using
-            // pre-evaluated paramaters, not doing new evaluations
-            allResponses={{}}
-          />
+        {message ? (
+          <>
```

Comment:
I think this is causing a weird problem : (

Parts of the History gets hidden by mysterious blank area
<img width="937" alt="Screen Shot 2022-11-11 at 9 49 13 AM" src="https://user-images.githubusercontent.com/16461988/201203304-e375e72e-26ec-4d73-b3a2-1ba29a637273.png">

<img width="938" alt="Screen Shot 2022-11-11 at 9 49 19 AM" src="https://user-images.githubusercontent.com/16461988/201203284-48105ec7-2d47-41a4-af03-dfada808bce5.png">

Reply:
Ok - after running another `migrate` and reopening the weird blank area stopped showing and instead I've noticed this blinking button on hover... Not sure what's going on.


https://user-images.githubusercontent.com/16461988/201203991-7338db44-a4ba-4f01-bf5f-2518ac350bc4.mov